includes/options/views/class-amp-analytics-options-serializer.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+require_once( AMP__DIR__ . '/includes/utils/class-amp-html-utils.php' );
+
+class Analytics_Options_Serializer {
+
+	public static function save() {
+
+		if ( is_admin() ) {
```

Comment:
This check prevent the checks to be run when the code is invoked from the test suite. Is there other way to prevent the tests from breaking (without the check was getting a permission error when running test_amp-analytics_options.php)